terraform/modules/data-share-service/ec2.tf

```diff
@@ -12,11 +12,12 @@ data "aws_ami" "amazon_linux" {
 }
 
 resource "aws_instance" "rds_bastion_host" {
-  ami                    = data.aws_ami.amazon_linux.id
-  instance_type          = "t3a.nano"
-  key_name               = aws_key_pair.rds_bastion_key_pair.key_name
-  subnet_id              = module.vpc.public_subnet_ids[0]
-  vpc_security_group_ids = [aws_security_group.rds_bastion_host_sg.id]
+  ami                         = data.aws_ami.amazon_linux.id
+  instance_type               = "t3a.micro"
```

Comment:
Why the size bump?

Reply:
Ah yes, I should have added a comment here. The link on the ticket recommended this size - I wasn't sure if it was necessary or not